renderdoc/replay/replay_driver.cpp

```diff
@@ -565,7 +565,7 @@ bool HighlightCache::FetchHighlightPositions(const MeshDisplay &cfg, FloatVector
   {
     uint32_t dim = PatchList_Count(cfg.position.topo);
 
-    uint32_t v0 = uint32_t(idx / dim) * dim;
+    uint32_t v0 = dim == 0 ? 0 : (uint32_t(idx / dim) * dim);
```

Comment:
Where does the division by zero still happen? `PatchList_Count` is called from inside an if that means `topo >= Topology::PatchList`, so the lowest return value possible should then be 1. Even if the input is invalid and somehow higher than `PatchList_32CPs` wouldn't it return a greater positive value? Is there some code elsewhere that is calling it for non-patchlist topologies and then dividing by the result?